invenio_communities/config.py

```diff
@@ -23,7 +23,12 @@
 
 import pkg_resources
 
-COMMUNITIES_REQUEST_EXPIRY_TIME = 7  # Default expiry time delta in days
+COMMUNITIES_REQUEST_EXPIRY_TIME = 7
+"""Time after which the inclusion requests automatically expire (in days)."""
+
+COMMUNITIES_DELETE_HOLDOUT_TIME = 7
+"""Time after which the communities marked for deletion are """
+"""hard-deleted from the system (in days)."""
```

Comment:
`"""` works on multiple lines, hence starting new line with `"""` is not necessary.